framework/route.py

```diff
@@ -2,20 +2,20 @@
 import functools
 
 # External Libraries
-from flask import request
+from quart import request
 
 # Sayonika Internals
 from framework.sayonika import Sayonika
 
 __all__ = ("route", "Route", "multiroute")
 
 
-def route(path, **kwargs):
+async def route(path, **kwargs):
```

Comment:
decorators shouldnt be async, only the inner functions. Please refactor this to ensure it works. This can be verified by testing all routes.